tests/test_transforms/test_inference/test_inverse_transform.py

```diff
@@ -147,6 +148,12 @@ def _test_inverse_transform_train_subset_segments(self, ts, transform, segments)
             # math
             (AddConstTransform(in_column="target", value=1, inplace=False), "regular_ts"),
             (AddConstTransform(in_column="target", value=1, inplace=True), "regular_ts"),
+            (
```

Comment:
In some test cases your `out_column` is `target` in some it is `positive`. You also don't check that we have a logic for `inverse_transform` in case of `out_column not in (left_column, right_column)`?

Reply:
Because it raise ValueError in case if it is inverse_transform and out_column is not left_column or right_column

Reply:
We will discuss this above.